internal/fuzzbinder/fuzzscraper.go

```diff
@@ -366,28 +381,48 @@ func GetFuzzTargets(fuzzParameters *FuzzParameters, date string) ([]string, erro
 
 // GetEvidences gets the list of the evidence files used by the fuzzscraper.
 // The expected date format is "YYYYMMDD".
-func GetEvidences(date string, fuzztargets []string, fuzzParameters *FuzzParameters) map[string]string {
-	evidences := make(map[string]string)
+func GetEvidences(date string, fuzzTargets []string, fuzzParameters *FuzzParameters) ([]amber.ClaimEvidence, error) {
+	evidences := make([]amber.ClaimEvidence, len(fuzzTargets)+2)
```

Comment:
Why is a fixed-size array declared here? What is the benefit of it?

Reply:
I just added it to avoid linting error `Consider pre-allocating ...` but I modified it slightly to make it nicer.